src/extension.ts

```diff
@@ -19,15 +19,20 @@ export async function activate(ctx: vscode.ExtensionContext): Promise<void> {
   //
   // This is janky, but that's alright since it provides such minimal
   // functionality to the extension.
-  const remoteSSHExtension = vscode.extensions.getExtension("ms-vscode-remote.remote-ssh")
-  if (!remoteSSHExtension) {
+  const openRemoteSSHExtension = vscode.extensions.getExtension("anysphere.open-remote-ssh")
+  // If the "anysphere.open-remote-ssh" extension is available, it is used with priority
+  // If it is not found, then the extension "ms-vscode-remote.remote-ssh" is sought as a fallback.
```

Comment:
Nit: Let's add a comment to explicitly mention this is for non-official VS Code distributions. Just so in the future we know!